unicoremc/models.py

```diff
@@ -586,8 +577,25 @@ def get_marathon_app_data(self):
             "labels": {
                 "domain": domain,
                 "country": self.get_country_display(),
-                "project_type": self.application_type.project_type,
-                "staticfiles_path": self.get_staticfiles_path(),
+                "project_type": self.application_type.project_type
+            },
+            "container": {
+                "type": "DOCKER",
+                "docker": {
+                    "image": "universalcore/%s"
+                        % self.application_type.get_qualified_name(),
+                    "forcePullImage": True,
+                    "network": "BRIDGE",
+                    "portMappings": [{"containerPort": 8080, "hostPort": 0}],
```

Comment:
That's the port running inside the container. The `"hostPort": 0` tells marathon to portforward to port `8080` inside the container and assign a random port to the host. So a running app will end up with the SSH equivalent `ssh dockercontainer -L 35691:localhost:8080`